kube/manifest/do-migration/migration-seed-aws-pod.yaml

```
@@ -0,0 +1,149 @@
+    echo "Update package manager..."
+    apt-get update
+    
+
+    echo "Install rsync, SSH, etc..."
+    apt-get install -y rsync openssh-server netcat-openbsd
+    
+    # 
+    echo "Config SSH service"
+    mkdir -p /var/run/sshd
+    echo 'root:password123' | chpasswd
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)